File: web/src/main/java/com/tonitealive/server/web/security/SpringSecurityWebAppConfig.java
package com.tonitealive.server.web.security;

import org.springframework.context.annotation.Configuration;
import org.springframework.security.config.annotation.web.builders.HttpSecurity;
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;

import static com.stormpath.spring.config.StormpathWebSecurityConfigurer.stormpath;

@Configuration
public class SpringSecurityWebAppConfig extends WebSecurityConfigurerAdapter {

    @Override
    protected void configure(HttpSecurity http) throws Exception {
        http.apply(stormpath());
    }

}

File: src/main/kotlin/com/tonitealive/server/SpringSecurityWebAppConfig.kt
package com.tonitealive.server

import com.stormpath.spring.config.StormpathWebSecurityConfigurer.stormpath
import org.springframework.context.annotation.Configuration
import org.springframework.security.config.annotation.web.builders.HttpSecurity
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter

@Configuration
open class SpringSecurityWebAppConfig : WebSecurityConfigurerAdapter() {

    override fun configure(http: HttpSecurity?) {
        http?.apply(stormpath())?.and()?.csrf()?.disable()
    }

}